utilities/config/hostname_unix.go

```diff
@@ -20,9 +23,43 @@ func getFQDNLinux() (string, error) {
 	cmd := exec.Command("/bin/hostname", "-f")
 	output, err := cmd.CombinedOutput()
 	if err != nil {
-		return "", errors.Wrap(err, constants.GetFQDNLinuxError)
+		// if command line doesn't work, try to look up by IP
+		fqdn, err := getFQDNByIP()
+		if err != nil {
+			return "", errors.Wrap(err, constants.GetFQDNLinuxError)
+		}
+		return fqdn, nil
 	}
 	fqdn := string(output)
 	fqdn = fqdn[:len(fqdn)-1]
 	return fqdn, nil
 }
+
+func getFQDNByIP() (string, error) {
+	// thanks for https://github.com/ShowMax/go-fqdn
```

Comment:
Can we just vendor and import go-fqdn?  That was we have proper LICENSE and attribution?